tests/test_app.py

```diff
@@ -34,13 +54,53 @@ def test_get_valid_title_path(self, mock_data):
     @mock.patch('service.server.get_title_register', return_value=DN1000_title)
     def test_get_correct_title_number(self, mock_data):
         response = self.app.get('/titles/DN1000')
-        self.assertTrue(str('"title_number": "DN1000"') in str(response.data))
+        assert str('"title_number": "DN1000"') in response.data.decode()
```

Comment:
I realise it's just refactoring. We could get rid of `str(...)` around this string, though. There's a similar thing in the test below.